ci-operator/step-registry/hypershift/aws/destroy-nested-management-cluster/hypershift-aws-destroy-nested-management-cluster-chain.yaml

```diff
@@ -52,7 +53,8 @@ chain:
       --name="${CLUSTER_NAME}" \
       --namespace="${HYPERSHIFT_NAMESPACE}" \
       --infra-id=${CLUSTER_NAME} \
-      --base-domain=${DOMAIN}
+      --base-domain=${DOMAIN} \
+      --cluster-grace-period 40m
```

Comment:
we've noticed a lot of failures lately that were caused by a [cluster-destroy step taking more than 10 minutes](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/40321/rehearse-40321-pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests/1668900603004194816/artifacts/appstudio-hac-e2e-tests/destroy-management-cluster/build-log.txt).

this change is based on [settings in hosted-cluster wf](https://github.com/openshift/release/blob/c0a7ee63273d5f4525534732d3831f736cb51294/ci-operator/step-registry/hypershift/hostedcluster/destroy/hostedcluster/hypershift-hostedcluster-destroy-hostedcluster-commands.sh#L41)